inferred-spans/src/main/java/io/opentelemetry/contrib/inferredspans/internal/semconv/Attributes.java

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.contrib.inferredspans.internal.semconv;
+
+import io.opentelemetry.api.common.AttributeKey;
+
+public class Attributes {
+
+  private Attributes() {}
+
+  public static final AttributeKey<String> CODE_STACKTRACE =
+      AttributeKey.stringKey("code.stacktrace");
+  public static final AttributeKey<Boolean> LINK_IS_CHILD = AttributeKey.booleanKey("is_child");
+  public static final AttributeKey<Boolean> SPAN_IS_INFERRED =
+      AttributeKey.booleanKey("is_inferred");
```

Comment:
`is_child` and `is_inferred` are feature-specific attributes which I don't see as candidated for SemConv.
Is there a best-practice for namespacing such attributes? E.g. to use the feature name as prefix (`inferred_spans.`)?